src/schnorr.rs

```diff
@@ -12,46 +13,56 @@ use crate::{
 
 #[allow(non_snake_case)]
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
-/// ID type which encapsulates the ID and a schnorr proof of ownership of the polynomial
+/// Encapsulatetion of the ID and a zero knowledge proof of ownership of a private key bound to ID
 pub struct ID {
-    /// The ID
+    /// ID
     pub id: Scalar,
-    /// The public schnorr response
-    pub kG: Point,
-    /// The aggregate of the schnorr committed values
-    pub kca: Scalar,
+    /// Commitment to the proof random value
+    pub R: Point,
+    /// Sigma protocol response
+    pub s: Scalar,
```

Comment:
If we're going to change the field names here, let's give them more descriptive names like `random_commitment` and `sigma` or whatever else would be more informative. It also makes it a bit more easier to follow if someone comes in with an understanding of the math but is used to different notation for the underlying values.

Reply:
All `sigma` protocols have three phases: `commitment`, `challenge`, and `response`, where you can use either commitment+response or challenge+response as the proof.  There seems to be a gradual consensus towards `R, z` (FROST), `r, s` (Taproot), or the generic `R, s` when using commitment+response.  

I really don't care what we do, as long as we change it.  The current comments are just wrong, and the variable names are extremely confusing.  What did you end up doing in the `sBTC` protobufs?  `random_commitment` and `sigma_response`?

Reply:
Looks like the `sBTC` protobufs used the incorrect comments as the basis of the variable names, so let's not do that.  I'll go with `random_commitment` and `sigma_response`, unless you think `commitment` and `response` is descriptive enough.

Reply:
Yeah, let's go with `random_commitment` and `sigma_response`.

> Looks like the `sBTC` protobufs used the incorrect comments as the basis of the variable names

Ooops, yeah we'll fix that when we update the wsts dependency (or sooner).

Reply:
Done